<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calc</class>
 <widget class="QMainWindow" name="Calc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>610</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>610</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>610</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calc</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="exprLine">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>700</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>760</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>35</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <stylestrategy>NoAntialias</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="sinButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SIN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cosButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>COS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tanButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TAN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>LOG</string>
    </property>
   </widget>
   <widget class="QPushButton" name="oneButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="twoButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="threeButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>60</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="acosButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ACOS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sixButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>160</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fiveButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>160</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fourButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>160</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lnButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>LN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="atanButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ATAN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="asinButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ASIN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minusButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nineButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>260</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eightButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>260</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sevenButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>260</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="divButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="multButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plusButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="powButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>360</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>360</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zeroButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>360</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>450</y>
      <width>100</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CLEAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeBrackButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>360</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openBrackButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>360</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sqrtButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SQRT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resultButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>450</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>40</pointsize>
      <italic>true</italic>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dotButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>360</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="xButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>460</y>
      <width>100</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>460</y>
      <width>100</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MOD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="graphButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>460</y>
      <width>141</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>GRAPH</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>470</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X value</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="xValue">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>490</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
